### PR TITLE
Prepare for release of VSCode 2.1.0

### PR DESCRIPTION
**Pull request by limemloh**
## Purpose

Prepare a release of the VSCode extension with the newest version of cargo-concordium (3.1.4)

### CODE REVIEW

File: concordium-smart-contract-testing/Cargo.lock [hunk 1]
```diff
@@ -0,0 +1,1705 @@
+# This file is automatically @generated by Cargo.
```
Comment (abizjak) [inline concordium-smart-contract-testing/Cargo.lock:1]: What is this here for?

Comment (abizjak) [reply concordium-smart-contract-testing/Cargo.lock:1]: This project was removed from this repository.

**Review by abizjak — APPROVED** [review]
Please remove the lock file. Otherwise looks good.

File: concordium-smart-contract-testing/Cargo.lock (hunk 1 above)
Comment (limemloh) [reply concordium-smart-contract-testing/Cargo.lock]: ups

**Pull request merged**